django_mailbox/models.py

```diff
@@ -361,6 +368,24 @@ def _process_message(self, message):
             msg.to_header = utils.convert_header_to_unicode(
                 message['Delivered-To']
             )
+        #rulzart - Save the email date
```

Comment:
I do not know if it's a good place for signature by. It can confuse. Commits are marked with authorship.